sphinx_wagtail_theme/footer.html

```diff
@@ -13,7 +13,7 @@
                 </nav>
             {% endif %}
             <div class="text-center">
-                {%- if theme_show_last_updated and last_updated %}
+                {%- if last_updated %}
                 <p>Last rendered: {{ last_updated|e }}</p>
```

Comment:
While we're near here, the choice of "rendered" here strikes me as odd and potentially confusing. Maybe change it?

```suggestion
                <p>Last updated: {{ last_updated|e }}</p>
```